tools/PowerShell-DSC/DatadogApmDotnet.ps1

```diff
@@ -0,0 +1,60 @@
+Configuration DatadogApmDotnet
+{
+    param
+    (
+        # Target nodes to apply the configuration
+        [string]$NodeName = 'localhost',
+
+        # Determines whether to install the Agent
+        [bool]$InstallAgent = $true,
+
+        # Determines whether to install the Tracer
+        [bool]$InstallTracer = $true
+    )
+
+    Import-DscResource -ModuleName PSDscResources -Name MsiPackage
+    Import-DscResource -ModuleName PSDscResources -Name Environment
+
+    # Version of the Agent package to be installed
+    $AgentVersion = '7.27.0'
+
+    # Version of the Tracer package to be installed
+    $TracerVersion = '1.26.1'
```

Comment:
Can you add this to `SetAllVersions.cs`?

Reply:
I was still figuring things out and I wasn't sure if we wanted to version this file like that. Another option could be to have a DSC Configuration for each version:
```
DatadogApmDotnetTracer-1.26.1.ps1
DatadogApmDotnetTracer-1.27.0.ps1
```
(but then we probably want to split the Agent into a separate Configuration with it's own version? rabbit hole...)

The other issue is that updating this and tagging the `master` branch is not correct, since we still need to update the product id in this file _after_ building the msi. Not sure what to do about that either.

Reply:
I'm not sure what happens if a Configuration requires one msi at one point in time, and later the same Configuration requires a different msi. Does one Config replace the other, or does the machine stay in a unstable state trying to apply _both_ Configs, but applying one undoes the other?

Reply:
If we could update the product id and tag the commit with the correct version, then users could access the versioned files by tag?
https://github.com/DataDog/dd-trace-dotnet/blob/v1.26.1/tools/PowerShell-DSC/DatadogApmDotnet.ps1
(404, don't try it)